apps/desktop/src/app/components/passkeys.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component } from "@angular/core";
+import { Router } from "@angular/router";
+
+import { DesktopSettingsService } from "../../platform/services/desktop-settings.service";
+
+@Component({
+  standalone: true,
+  template: `
+    <div
+      style="background:white; display:flex; justify-content: center; align-items: center; flex-direction: column"
+    >
+      <h1>Select your passkey</h1>
+      <br />
+      <button bitButton type="button" buttonType="secondary" (click)="closeModal()">Close</button>
+    </div>
+  `,
+})
+export class PasskeysComponent {
```

Comment:
suggestion: match naming convention on browser, see more info here: https://contributing.bitwarden.com/architecture/deep-dives/passkeys/naming-convention

```suggestion
export class Fido2Component {
```